### ISSUE TITLE
Support ownCloud/Nextcloud Music App

### ISSUE THREAD

**Issue by sumnerevans** [sublime-music/sublime-music-python#327]
In GitLab by @dadainfo on Nov 9, 2021, 07:14

Hey there,

I'm a massive user of ownCloud [Music App](https://github.com/owncloud/music). This application is compatible with subsonic API but Sublimes fails when I try to add my server.

Here is the error :

```
2021-11-09 10:59:44,695:WARNING:root:adapter:Salted auth not supported, falling back to regular password auth
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.8/site-packages/sublime_music/app.py", line 167, in do_activate
    self.show_configure_servers_dialog()
  File "/home/user/.local/lib/python3.8/site-packages/sublime_music/app.py", line 1019, in show_configure_servers_dialog
    self.app_config.save()
  File "/home/user/.local/lib/python3.8/site-packages/sublime_music/config.py", line 234, in save
    json = self.to_json(indent=2, sort_keys=True)
  File "/home/usef/.local/lib/python3.8/site-packages/dataclasses_json/api.py", line 49, in to_json
    return json.dumps(self.to_dict(encode_json=False),
  File "/home/user/.local/lib/python3.8/site-packages/dataclasses_json/api.py", line 85, in to_dict
    return _asdict(self, encode_json=encode_json)
  File "/home/user/.local/lib/python3.8/site-packages/dataclasses_json/core.py", line 346, in _asdict
    return _encode_overrides(dict(result), _user_overrides_or_exts(obj),
  File "/home/user/.local/lib/python3.8/site-packages/dataclasses_json/core.py", line 110, in _encode_overrides
    v = encoder(v) if encoder is not None else v
  File "/home/user/.local/lib/python3.8/site-packages/sublime_music/config.py", line 71, in encode_providers
    return {
  File "/home/user/.local/lib/python3.8/site-packages/sublime_music/config.py", line 74, in <dictcomp>
    "ground_truth_adapter_type": config["ground_truth_adapter_type"].__name__,
TypeError: 'ProviderConfiguration' object is not subscriptable
```

I already opened [an issue](https://github.com/owncloud/music/issues/915#issuecomment-964046216) on Music's Github.

I tried Sublime on Ubuntu 20.04 via pip installation method.

Regards

**Comment by sumnerevans** [comment]
marked this issue as related to #320

**Comment by sumnerevans** [comment]
In GitLab by @dadainfo on Nov 12, 2021, 01:36

Fixed with v0.11.16 

Thanks !
